ports/atmel-samd/mpconfigport.mk

```diff
@@ -65,19 +65,19 @@ CIRCUITPY_SAMD = 1
 endif
 
 ifndef CIRCUITPY_ULAB
-ifneq ($(CIRCUITPY_SMALL_BUILD),1)
+ifneq ($(CIRCUITPY_FULL_BUILD),0)
 CIRCUITPY_ULAB = 1
 endif
 endif
 
 ifndef CIRCUITPY_RGBMATRIX
-ifneq ($(CIRCUITPY_SMALL_BUILD),1)
+ifneq ($(CIRCUITPY_FULL_BUILD),0)
 CIRCUITPY_RGBMATRIX = 1
 endif
```

Comment:
This is the same as `CIRCUITPY_RGBMATRIX = $(CIRCUITPY_FULL_BUILD)`

Reply:
RGBMatrix is only enabled for the SAMD, so this is appropriate here. If it was full build, everything else would need to turn it off.

Reply:
I was just saying that the code
```
ifneq ($(CIRCUITPY_FULL_BUILD),0)
CIRCUITPY_RGBMATRIX = 1
endif
```
is equivalent to the shorter
```
CIRCUITPY_RGBMATRIX = $(CIRCUITPY_FULL_BUILD)
```

Reply:
CIRCUITPY_FULL_BUILD is either 0 or 1, so you can just set the variable to its value.

Reply:
Oh you mean just here in this file, I see, it was the other way because it was setting the opposite of SMALL_BUILD but now that it's FULL_BUILD it's just one line. Changed.

Reply:
Sorry, I thought you meant you wanted it relocated.